### ISSUE TITLE
Feature request: add support for HTML and PHP

### ISSUE THREAD

**Issue by digisax** [kantlove/vscode-todo-parser#69]
It would be super helpful if support for HTML and PHP could be added to this extension.

**Comment by jdittrich** [comment]
Yes, I wonder how hard it is to add a new language (I'd like to have HTML and Markdown). Maybe @kantlove could give us a pointer what needs to be done?
